AppCenter/AppCenter/Internals/DelegateForwarder/MSAppDelegateForwarder.h

```diff
@@ -0,0 +1,20 @@
+#import "MSCustomApplicationDelegate.h"
+#import "MSDelegateForwarder.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Enum used to represent all kind of executors running the completion handler.
+ */
+typedef NS_OPTIONS(NSUInteger, MSCompletionExecutor) {
```

Comment:
Why are we using an enum but setting its values like flags?  Are these flags or values?  I can't tell.

Reply:
It's bitmask thanks to `NS_OPTIONS`, I can update the description.

Reply:
I wrote this comment before I knew about the quirks of Common Schema, I think this is just a weird implementation that we'll have to deal with